dev/wlp-jakartaee-transform/rules/jakarta-versions.properties

```diff
@@ -1,4 +1,3 @@
-com.ibm.wsspi.el=[4.0,5)
```

Comment:
Transformations are not longer necessary as this package is now used internally.